java/memory/memory-core/src/test/java/org/apache/arrow/memory/JavaForeignAllocationManager.java

```diff
@@ -0,0 +1,77 @@
+/*
```

Comment:
This was added as a hack to test the memory-core module with the FFM APIs. It will be useful for testing later, but can be ignored otherwise.